h2o-py/tests/testdir_algos/glm/pyunit_PUBDEV_7645_lambda_search_warning.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import print_function
+from builtins import range
+import sys, os
+sys.path.insert(1,"../../../")
+import h2o
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+from tests import pyunit_utils
+
+try:    # redirect python output
+    from StringIO import StringIO  # for python 3
+except ImportError:
+    from io import StringIO  # for python 2
+
+# This method test to make sure when a user set lambda_search=True and choose a lambda value,  a warning message 
+# should be generated and passed to the user.
+def test_benign():
```

Comment:
is this a type or does `benign` has some meaning?

Reply:
tumor type not cancerous.

Reply:
benign (good) or malignant (bad)